src/python/pants/engine/internals/parser.py

```diff
@@ -107,7 +107,13 @@ def __call__(self, **kwargs: Any) -> TargetAdaptor:
                 parse_state.add(target_adaptor)
                 return target_adaptor
 
-        symbols: dict[str, Any] = dict(object_aliases.objects)
+        class Cwd:
+            """Lets you call `cwd()` in a BUILD file to get the relative path."""
+
+            def __call__(self) -> str:
+                return parse_state.rel_path()
+
+        symbols: dict[str, Any] = {**object_aliases.objects, "cwd": Cwd()}
```

Comment:
This seems like alot of ceremony for what could be `"cwd": parse_state.rel_path`.

Reply:
Much better. Thanks!